poofarmer/.mono/metadata/ide_messaging_meta.txt

```diff
@@ -0,0 +1,2 @@
+﻿60822
+C:/Users/Leith/Downloads/Godot/Godot_v3.5.1-stable_mono_win64/Godot_v3.5.1-stable_mono_win64.exe
```

Comment:
We actually need to delete this file from the branch so we don't have problems once we pull down with the gitignore

Reply:
So like

`git rm --cached *.mono*`

I believe